schema/annotation-source.yaml

```diff
@@ -26,16 +26,6 @@ $defs:
         type: string
         description: >-
           A free-text description of the InformationEntity.
-      confidence_level:
-        $ref_curie: gks.core:Coding
-        description: >-
-          A coded term describing the strength of support that the information the statement
-          represents is true.
-      confidence_score:
-        $ref: "#/$defs/DataItem"
-        description: >-
-          A quantitative score reflecting the degree of confidence that the information 
-          the information entity represents is true.
```

Comment:
Removes confidence_level and confidence_score from InformationEntity (per discussion [here](https://github.com/ga4gh/va-spec/issues/84)) . . . they will be defined directly on the Statement class.